packages/xo-web/src/xo-app/vm/tab-advanced.js

```diff
@@ -1042,6 +1090,77 @@ export default class TabAdvanced extends Component {
                     </td>
                   </tr>
                 )}
+                {vm.boot.firmware === 'uefi' && (
+                  <tr>
+                    <th>{_('secureBoot')}</th>
+                    <td>
+                      <Toggle value={vm.secureBoot} onChange={value => editVm(vm, { secureBoot: value })} />
+                      <a
+                        className='text-muted'
+                        href='https://xcp-ng.org/docs/guides.html#guest-uefi-secure-boot'
+                        rel='noreferrer'
+                        style={{ display: 'block' }}
+                        target='_blank'
+                      >
+                        <Icon icon='info' /> {_('secureBootLinkToDocumentationMessage')}
+                      </a>
+                    </td>
+                  </tr>
+                )}
+                {vm.boot.firmware === 'uefi' && semver.satisfies(host?.version, '>=8.3.0') && (
+                  <tr>
+                    <th>{_('secureBootStatus')}</th>
+                    <td>
+                      {SECUREBOOT_STATUS_MESSAGES[vmSecurebootReadiness]}
+                      {(vmSecurebootReadiness === 'setup_mode' ||
+                        vmSecurebootReadiness === 'certs_incomplete' ||
+                        vmSecurebootReadiness === 'ready_no_dbx') &&
+                        host?.productBrand === 'XCP-ng' && (
+                          <a
+                            className='text-warning'
+                            /* href=link to be provided by Samuel  */
+                            rel='noreferrer'
+                            style={{ display: 'block' }}
+                            target='_blank'
+                          >
+                            <Icon icon='alarm' /> {_('secureBootLinkToDocumentationMessage')}
+                          </a>
+                        )}
+                    </td>
+                  </tr>
+                )}
+                {vm.boot.firmware === 'uefi' && (
+                  <tr>
+                    <th>{_('propagateCertificatesTitle')} </th>
+                    <td>
+                      <ActionButton
+                        btnStyle='primary'
+                        disabled={isDisabled}
+                        handler={this._confirmUefiMode}
+                        icon='vm-clone'
+                      >
+                        {_('propagateCertificates')}
+                      </ActionButton>
+                      {isDisabled && (
```

Comment:
If I read it well, this condition is not enough. It should not display the warning about UEFI being required in the case when the reason why it's disabled is because `poolGuestSecurebootReadiness === 'not_ready'`.

As currently implemented, when the pool is not ready for Secure Boot, both warnings are displayed. We should only see one.

![image](https://github.com/vatesfr/xen-orchestra/assets/451043/e1ad9f6e-c405-4a80-be11-88bf191b39ac)